sample/CustomerApi/Helpers/OptionallyPatchedExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using LogOtter.HttpPatch;
+
+namespace CustomerApi;
+
+public static class OptionallyPatchedExtensions
+{
+    public static bool IsIncludedInPatchAndIsDifferentFrom<T>(this OptionallyPatched<T> optionallyPatched, T value)
```

Comment:
@PeterStephenson Do you think this is worth including in `HttpPatch`?

Reply:
As discussed, resolving for now so we can do a release, as this is a non-breaking suggestion